morango/utils/register_models.py

```diff
@@ -3,16 +3,30 @@
 like to inherit from to make their data syncable. This method takes care of registering morango
 data structures on a per-profile basis.
 """
-import six
 
+from collections import namedtuple
 from django.apps import apps
 from django.db import models
-from morango.models import AbstractStoreModel, SyncableModel
-from morango.constants import buffer_kinds, morango_structures
+from django.utils import six
+from morango.models import AbstractStoreModel, AbstractDatabaseMaxCounter, SyncableModel
+
 from .controller import _profiles
 
+Profile = namedtuple('Profile', 'partitions buffer store database_max_counter')
+
+
+def register_morango_profile(profile="", partitions=(), module=""):
```

Comment:
Easiest way to indicate that these aren't optional arguments would be to not have default values here.